### ISSUE TITLE
From a technical standpoint, how does Apple's App Transparency Framework work?

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/gpc-android#34]
@kalicki1, can you take the lead on this? And please contribute here as well, @bella-tassone and @ekuller. 

Today is the official release date of iOS 14.5 with the [App Tracking Transparency Framework](https://developer.apple.com/documentation/apptrackingtransparency). Apple made a short [video](https://www.youtube.com/watch?v=Ihw_Al4RNno) about it that is quite good. But it stops short of explaining how it works. It is not 100% clear to me how it is ensured technically that a user is not being tracked. Sure, as a user, I can select not being tracked, but then what?

As a developer, how do I know that a user has enabled App Tracking Transparency? How is this information made available to me, if at all? Apart from implementing the Framework in my app's codebase, do I need to set up something on my backend? Is my app automatically blocked by the Framework to send out HTTP requests to ad networks if a user enables it?

In short, from a technical standpoint, how does Apple's App Transparency Framework work? How does it technically ensure that a user who turned it on is not tracked.

The answer to that question will be informative for our own approach.
